### ISSUE TITLE
Provide capability to track unhandled exceptions in Application Insights

### ISSUE THREAD

**Issue by tomkerkhove** [arcus-azure/arcus.webapi#16]
Provide capability to track unhandled exceptions in Application Insights.

This depends on #14.

**Comment by tomkerkhove** [comment]
This is actually already done via #14 which uses `ILogger` - Can you confirm this @fgheysels?

**Comment by fgheysels** [comment]
> This is actually already done via #14 which uses `ILogger` - Can you confirm this @fgheysels?

Yes this is done by the functionality mentionned in #14.  One of my pull-requests mentionned #16 as well, so this one can be closed imo.

**Comment by tomkerkhove** [comment]
Perfect, thanks @fgheysels !
